src/Microsoft.Health.Fhir.Api/Features/Routing/ServerSideRequestForgeryException.cs

```diff
@@ -0,0 +1,42 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Api.Features.Routing
+{
+    /// <summary>
+    /// Represents an exception that is thrown when a user engages in behavior that would make others
+    /// suspectible to a Server-Side Request Forgery (SSRF) attack.
+    /// </summary>
+    public sealed class ServerSideRequestForgeryException : FhirException
```

Comment:
What is creating this exception? I see where it is handled, but not where it is made.

Reply:
I will be adding middleware in wp since antissrf package is only internal. I initially thought of adding the middleware in OSS itself but then noticed, nuget package is available internally only. For reference the OSS PR that I am going to close and add middleware in wp https://github.com/microsoft/fhir-server/pull/4848/files#diff-a2053d6bf1ad035351687ac5724990005a560a7ae01b7a13c3625af1d26d0dbc